src/System.Data.SqlClient/tests/ManualTests/SQL/MARSTest/MARSTest.cs

```diff
@@ -14,6 +14,7 @@ public static class MARSTest
         private static readonly string _connStr = (new SqlConnectionStringBuilder(DataTestUtility.TcpConnStr) { MultipleActiveResultSets = true }).ConnectionString;
 
         [CheckConnStrSetupFact]
+        [PlatformSpecific(TestPlatforms.Windows)]
```

Comment:
I branched off the previous PR. I expected that Git would match the commits and not show the older difference in this PR. 
Rebased with dotnet/master and updated this PR.